hangman.rb

```diff
@@ -0,0 +1,194 @@
+require "Colorize"
+
+class Hangman
+
+  def initialize
+    # List of possible game words. They are randomly selected during initialize.
+    @word_list =  ["w h a l e", "f l a g", "d o g", "f o r e s t",
+    "b r a v e", "r o s e", "g i f t", "s t a y", "a x", "d o l p h i n"]
+
+    # ASCII representations of the hangman graphics
+    @hanger_start   = """
+    |     _________
+    |     |/      |
+    |     |
+    |     |
+    |     |
+    |     |
+    |     |
+    | ____|___
+    """
+    @hanger_head = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |
+    |     |
+    |     |
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+   @hanger_body = """
+
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |       |
+    |     |       |
+    |     |
+    |     |
+    | ____|___
+   """.colorize(:red)
+
+    @hanger_arm1 = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |       |\\
+    |     |       |
+    |     |
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+    @hanger_arm2 = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |      /|\\
+    |     |       |
+    |     |
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+    @hanger_leg1 = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |      /|\\
+    |     |       |
+    |     |      /
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+    @hanger_leg2 = """
+    |     _________
+    |     |/      |
+    |     |      (∞)
+    |     |      /|\\
+    |     |       |
+    |     |      / \\
+    |     |
+    | ____|___
+    """.colorize(:red)
+
+    @winner = """
+    HURAHH!!!! I'm alive!
+           (∞)
+           \\|/
+            |
+           / \\
+    """.colorize(:light_blue)
+
+    # This creates an array of possible ASCII outcomes as the user makes mistakes or wins
+    @hanger_list = [@hanger_head, @hanger_body, @hanger_arm1, @hanger_arm2, @hanger_leg1, @hanger_leg2, @winner]
+
+    start
+    answer
+    puts "Word: ".colorize(:light_blue) + blank
+    play
+  end
+
+  # Prompts the user to begin a new game
+  def start
+    puts "Let's play Hangman!"
+    puts "To start hit enter"
+    user = gets
+    if user.length == 1
+    puts @hanger_start
+    puts "Looking for a word for you... one sec!"
+    puts "To quit during gameplay, type 'quit'"
+    sleep 1
+    end
+  end
+
+  # Randomly generates answer word
+  def answer
+    @random_word = @word_list[rand(0...@word_list.length)]
+    #puts "#{@random_word}".colorize(:black)
+    return @random_word

```

Comment:
Returning the `@random_word` instance variable here seems unnecessary since you're calling this answer method on line 100 but not doing anything with the return values